tests/output/staging/10061-UNI-1355.cs

```diff
@@ -8,9 +8,9 @@ public override MyNodeType NodeType
         {
             if (entity != null)
                 return entity.ReadState == ReadState.Initial ?
-                       source.NodeType :
-                       entity.EOF ? MyNodeType.EndEntity :
-                       entity.NodeType;
+                    source.NodeType :
+                    entity.EOF? MyNodeType.EndEntity:

```

Comment:
Why the space removal ?

Reply:
Previously there are two tabs and expected is one tab according to unity config and changes we have done. So removed one tab space

Reply:
I was talking about the space on the ternary operator `?`